src/main/java/org/spdx/storage/listedlicense/LicenseJsonTOC.java

```diff
@@ -204,11 +205,11 @@ public void addLicense(SpdxListedLicense license, String licHTMLReference, Strin
 			boolean deprecated) throws InvalidSPDXAnalysisException {
 		LicenseJson lj = new LicenseJson();
 		lj.setDeprecatedLicenseId(deprecated);
-		lj.setDetailsUrl(licHTMLReference);
+		lj.setDetailsUrl(toAbsoluteURL(licJSONReference));
 		lj.setLicenseId(license.getId());
 		lj.setName(license.getName());
 		lj.setOsiApproved(license.isOsiApproved());
-		lj.setReference(licJSONReference);
+		lj.setReference(licHTMLReference);
```

Comment:
Let's make this absolute as well - even though it was previously relative, the HTML it is referencing should have been absolute from the beginning

Reply:
I was also wondering why only json had absolute URLs!
I agree with using absolute URLs, and now changed to absolute one.